contracts/modules/v2/StakingPool.sol

```diff
@@ -42,6 +42,17 @@ contract StakingPool is ERC20 {
     // uint48 _unused;
   }
 
+  struct AllocateCapacityParams {
+    uint productId;
+    uint coverAmount;
+    uint rewardsDenominator;
+    uint period;
+    uint capacityFactor;
```

Comment:
What is the difference between capacityFactor and capacityReductionRatio?

Reply:
If capacityFactor is a global param is it worth drilling it through params? It would also be more obvious what it does if we name it globalCapacityFactor so it won't be easily confused with capacityReductionRatio.

Reply:
i'll add comments and rename to globalCapacityFactor